deploy/template.yaml

```diff
@@ -64,11 +64,18 @@ Resources:
           ENVIRONMENT: !Sub "${Environment}"
           AUTH_CODES_TABLE_NAME: !Select [1, !Split ['/', !GetAtt AuthCodesTable.Arn]]
           ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt AccessTokensTable.Arn]]
+          ORCHESTRATOR_ID: !Sub "/${Environment}/clients/orchestrator-stub/id"
+          ORCHESTRATOR_ISSUER: !Sub "/${Environment}/clients/orchestrator-stub/issuer"
+          ORCHESTRATOR_CERTIFICATE_FOR_CORE_TO_VERIFY: !Sub "/${Environment}/clients/orchestrator-stub/publicCertificateForCoreToVerify"
+          ORCHESTRATOR_VALID_CALLBACK_URL: !Sub "/${Environment}/clients/orchestrator-stub/validCallBackUrls"
+          ORCHESTRATOR_AUTHENTICATION_METHOD: !Sub "/${Environment}/clients/orchestrator-stub/authenticationMethod"
```

Comment:
I think drop these for now. We'll probably just want to provide a path to the top level of the client config. Can do that in a later ticket though.